Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/AddDocStrucTypeDialog.java

```diff
@@ -0,0 +1,301 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.dataeditor;
```

Comment:
No, because in contrast to the editor in Production 2 this new version explicitely is not only used for editing metadata, but structural data as well.

Reply:
That was already possible with 2x. The name has **always** been wrong. I am glad that we finally take this into account.